package.json

```diff
@@ -24,7 +24,7 @@
   "scripts": {
     "preinstall": "./scripts/download-mapbox-gl-native-ios.sh 3.2.0",
```

Comment:
Done. I upgraded both the iOS and Android SDKs. They seem to work fine on a first glance. Let's merge this PR, please :D